loader/vk_loader_platform.h

```diff
@@ -279,6 +299,13 @@ static inline char *loader_platform_dirname(char *path) {
     return path;
 }
 
+static inline char *loader_platform_executable_path(char buffer[1024]) {
+    DWORD ret = GetModuleFileName( NULL, buffer, 1024);
+    if (ret == 0) return NULL;
+    if (ret > 1024) return NULL;
```

Comment:
There's a subtle bug here. `GetModuleFileName` never writes the null terminator, so if the length is greater than 1023, you'll get an invalid string. You have code to fix this on Linux, so you'll need it here and likely in the mac code.